kanidmd/src/lib/constants/acp.rs

```diff
@@ -126,6 +126,29 @@ pub const JSON_IDM_SELF_ACP_WRITE_V1: &str = r#"{
     }
 }"#;
 
+pub const JSON_IDM_PEOPLE_SELF_ACP_WRITE_MAIL_V1: &str = r#"{
+    "attrs": {
+        "class": ["object", "access_control_profile", "access_control_modify"],
+        "name": ["idm_people_self_acp_write_mail"],
+        "uuid": ["00000000-0000-0000-0000-ffffff000041"],
+        "description": ["Builtin IDM Control for self write of mail for people accounts."],
+        "acp_receiver": [
+            "{\"eq\":[\"memberof\",\"00000000-0000-0000-0000-000000000033\"]}"
+        ],
+        "acp_targetscope": [
```

Comment:
Should this duplicate `acp_targetscope` be here? 🤔

Reply:
No, it shouldn't but the second one will over-ride it.

Reply:
It's a copy-paste error, thanks for catching that @yaleman

Fixed, removed the empty one.